### DISCUSSION TITLE
Show notification when the score of a habit falls below a certain threshold

### DISCUSSION THREAD

**Discussion by falematte** [iSoron/uhabits#268] [Feature Requests]
It would be useful to be able to set an automatic notification when a certain habit goes under a certain threshold.

**Comment by iSoron** [comment]
Thank you for the suggestion. I think this is an interesting feature. We should just be careful not to show it too often. I think it only makes sense to show this notification if the habit had a reasonably good score in the recent past, but fell considerably.
